public/lessons/react/setting_up_react.md

```diff
@@ -1,185 +1,63 @@
-
-# Example HTML File Head
-
-```html
-  <head>
-    <meta charset="UTF-8" />
-    <title>Hello World</title>
-    <script src="https://unpkg.com/react@16/umd/react.development.js"></script>
-    <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
-    <script src="https://unpkg.com/babel-standalone@6.15.0/babel.min.js"></script>
-  </head>
-```
-
-# Local Package Installation Requirements
-
-You will need to have the following to include React as a local package in your HTML file.
-
-- A package manager like **NPM** or **Yarn**
-- A package bundler like **Webpack** or **Browserify**
-- A compiler like **Babel**
-
-# Local Package Installation Setup
-
-We will start with installing React using NPM
-
-```sh
-npm init -y
-npm install --save react react-dom
-```
-
-# Local Package Installation 1
-
-Download the React, ReactDOM, and Babel Scripts from the Content Delivery Network (CDN)
-
-### React Development
-- https://unpkg.com/react@16/umd/react.development.js
+1. Install Create React App **globally** `npm install -g creat-react-app`
+2. From the command line type `npx create-react-app app-name`
 
-### ReactDOM Development
-- https://unpkg.com/react-dom@16/umd/react-dom.development.js
+This will create a directory for the front end of your app called `app-name`. Inside that there will be a package.json file with all the scripts you need to create a react app, run it locally, and build a production version of that site.
 
-### Babel Standalone
-- https://unpkg.com/babel-standalone@6.15.0/babel.min.js
+# Directory Structure
 
-# Local Package Installation 2
+There are two main parts of the `create-react-app` directory
 
-Put the content of the scripts into a directory called `src` in the project root
+- the `src` folder
+- the `public` folder
 
-```shell
-mkdir js
-mv path/to/downloads/react.development.js ./src/
-mv path/to/downloads/react-dom.development.js ./src/
-mv path/to/downloads/babel.min.js ./src/
-```
+> If you create a production build with `npm run build` you will additionally have a `build` folder. More on that when we talk about serving React projects next week!
 
-# Local Package Installation 3
+# Core Concept: the `src` folder
 
-Create an `index.html` file and source all three files in the head
+The `src` folder is where the meat of your React project lives
 
-```html
-<html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Hello World</title>
-    <script src="src/react.development.js"></script>
-    <script src="src/react-dom.development.js"></script>
-    <script src="src/babel.min.js"></script>
-  </head>
-  <body>
-    <div id="output"></div>
-  </body>
-</html>
-```
+- Create a JS file for each component
+- Import components where needed
+- The `App.js` file lays out the whole page
+- `index.js` draws the content to the page
+- CSS for your project also goes in here
 
-# Local Package Installation 4
+# Core Concept: the `public` folder
```

Comment:
Breaking these parts up and explaining them is a great idea here, will you end up showing how to do the local package installation steps or just scratch that completely?

Reply:
I'm intending on just scrapping that completely. If we have time I might go over it, but I think it needlessly complicates an already complex topic.